prow/git/git.go

```diff
@@ -346,6 +351,27 @@ func (r *Repo) mergeWithMergeStrategySquash(commitlike string) (bool, error) {
 	return true, nil
 }
 
+func (r *Repo) mergeWithMergeStrategyMergeAndMessage(commitlike, message string) (bool, error) {
```

Comment:
Shouldn't `mergeWithMergeStrategyMerge` call this?

Reply:
To keep backward compability I created a new one.

Reply:
Err... it is an unexported method, so there is no possibility of breaking anything.